Index: Cinescope/.gitignore
===================================================================
diff --git a/Cinescope/.gitignore b/Cinescope/.gitignore
deleted file mode 100644
--- a/Cinescope/.gitignore	(revision 678377d03143c680fd302b0d45bf905b9e524042)
+++ /dev/null	(revision 678377d03143c680fd302b0d45bf905b9e524042)
@@ -1,31 +0,0 @@
-
-.idea/
-.vscode/
-
-
-.venv/
-venv/
-
-
-__pycache__/
-*.pyc
-*.pyo
-
-
-.DS_Store
-
-
-*.db
-
-
-*.xlsx
-
-
-*.log
-
-
-*.csv
-
-
-.token
-.env
\ No newline at end of file
